css/drawingboard.css

```diff
@@ -1,112 +1,113 @@
 .drawing-board {
-  text-align: center;
+    text-align: center;
 }
 
 canvas {
-  background: #fff;
-  display: block;
-  margin: auto;
-  border-radius: 5px;
-  border: 5px solid black;
-  cursor: url(../img/cursor.png), crosshair;
+    background: #fff;
+    display: block;
+    margin: auto;
+    border-radius: 5px;
+    border: 5px solid black;
+    cursor: url(../img/cursor.png), crosshair;
+    width: 50%;
 }
 
 .controls {
-  min-height: 60px;
-  margin: 0 auto;
-  width: 600px;
-  border-radius: 5px;
-  overflow: hidden;
+    min-height: 60px;
+    margin: 0 auto;
+    width: 600px;
```

Comment:
Would you mind changing this to make the colours and "New Color" button scalable for mobile too?

(ie `width: 100%`; instead of `width:600px;`)